parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -68,15 +73,27 @@ installNvidiaFabricManager() {
     done
 }
 
-installNvidiaContainerRuntime() {
+installNvidiaContainerToolkit() {
     MARINER_NVIDIA_CONTAINER_RUNTIME_VERSION="3.13.0"
-    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.13.5"
+    MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.15.0"
+
+    if [[ $OS_VERSION == "2.0" ]]; then
+      MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION="1.13.5"
+    fi
     
-    for nvidia_package in nvidia-container-runtime-${MARINER_NVIDIA_CONTAINER_RUNTIME_VERSION} nvidia-container-toolkit-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} nvidia-container-toolkit-base-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} libnvidia-container-tools-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} libnvidia-container1-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION}; do
+    for nvidia_package in nvidia-container-toolkit-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} nvidia-container-toolkit-base-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} libnvidia-container-tools-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION} libnvidia-container1-${MARINER_NVIDIA_CONTAINER_TOOLKIT_VERSION}; do
       if ! dnf_install 30 1 600 $nvidia_package; then
         exit $ERR_APT_INSTALL_TIMEOUT
       fi
     done
+
+    if [[ $OS_VERSION == "2.0" ]]; then
+      for nvidia_package in nvidia-container-runtime-${MARINER_NVIDIA_CONTAINER_RUNTIME_VERSION}; do
```

Comment:
nit: Is the for loop still needed here since there's only one package?

Reply:
Nope, not needed. Will update this

Reply:
Removed the for loop - [commit](https://github.com/Azure/AgentBaker/pull/4808/commits/25a310b918a7ff556f4b2dada6f661d243b0a38b)